src/PrestaShopBundle/Resources/config/services/core/security.yml

```diff
@@ -4,3 +4,8 @@ services:
 
   prestashop.core.crypto.hashing:
     class: 'PrestaShop\PrestaShop\Core\Crypto\Hashing'
+
+  prestashop.core.security.folder_protector.vendor:
+    class: 'PrestaShop\PrestaShop\Core\Security\HtaccessFolderProtector'
+    arguments:
+      - '%kernel.project_dir%/vendor/.htaccess'
```

Comment:
it's not really a template. template must be store in resources directory instead

Reply:
Moved into `app/Resources/security/.htaccess.dist`